.github/workflows/self-push.yml

```diff
@@ -20,37 +20,34 @@ env:
   OMP_NUM_THREADS: 8
   MKL_NUM_THREADS: 8
   PYTEST_TIMEOUT: 60
+  SIGOPT_API_TOKEN: ${{ secrets.SIGOPT_API_TOKEN }}
+  TF_FORCE_GPU_ALLOW_GROWTH: true
+  # TODO: Should we enable this?
+  # RUN_PT_TF_CROSS_TESTS: 1
```

Comment:
Current push CI don't run PT/TF cross tests. Guess it was because we use framework specific docker images.

Now we use `transformers-all-latest-gpu` (as scheduled CI), do you think it's good to enable it?

Reply:
Yes, would be good indeed